probability.ipynb

```diff
@@ -2966,6 +2937,34 @@
     "elimination_ask('Burglary', dict(JohnCalls=True, MaryCalls=True), burglary).show_approx()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Elimination ask optimizations\n",
```

Comment:
This should be in title form, with every other word capitalized: Elimination Ask Optimization